backend/src/main/java/com/wootech/dropthecode/service/TeacherService.java

```diff
@@ -122,5 +122,26 @@ public void updateAverageReviewTime(Long id, Long reviewTime) {
         teacher.updateReviewCountAndTime(reviewTime);
         save(teacher);
     }
+
+    @Transactional
+    public void updateTeacher(LoginMember loginMember, TeacherRegistrationRequest teacherRegistrationRequest) {
+        Map<String, Language> languageMap = languageService.findAllToMap();
+        List<Language> languages = findLanguageByNames(teacherRegistrationRequest.getTechSpecs(), languageMap);
+        List<Skill> skills = findSkillsByNames(teacherRegistrationRequest.getTechSpecs(), skillService.findAllToMap());
+        teacherRegistrationRequest.validateSkillsInLanguage(languageMap);
+
+        TeacherProfile teacher = findById(loginMember.getId());
+
+        teacherLanguageService.deleteAllWithTeacher(teacher);
+        teacherLanguageService.saveAllWithTeacher(languages, teacher);
+
+        teacherSkillService.deleteAllWithTeacher(teacher);
+        teacherSkillService.saveAllWithTeacher(skills, teacher);
+
+        teacher.setTitle(teacherRegistrationRequest.getTitle());
```

Comment:
```suggestion
        teacher.update(teacherRegistrationRequest.getTitle(), teacherRegistrationRequest.getContent(), teacherRegistrationRequest.getCareer());
```
어차피 모두 덮어쓴다면 이런식으로 한번에 하는 건 어떻게 생각하시나요?

Reply:
좋네요! 적용하겠습니다